Cython/Compiler/ExprNodes.py

```diff
@@ -7185,6 +7185,74 @@ def analyse_as_python_attribute(self, env, obj_type=None, immutable_obj=False):
                 error(self.pos,
                       "Object of type '%s' has no attribute '%s'" %
                       (obj_type, self.attribute))
+        else:
+            self.python_attribute_lookup_warnings(env)
+
+    def python_attribute_lookup_warnings(self, env):
```

Comment:
This method sometimes uses `return` and sometimes `return None`. The latter seems more appropriate (given that it otherwise returns an entry), but since the only usage ignores the return value, I wonder what the goal here is.